COMRS-37 Update "Other information"

tmvumbi2 opened the pull request:
Jira ticket: https://jembiprojects.jira.com/browse/COMRS-37

tmvumbi2 committed:
COMRS-37 Updates concepts needed for registration other information

This commit adds and updates concepts (for both English and French) required to populate the drop-downs under 'Other information' in the registration form

tmvumbi2 committed:
COMRS-37 Add person attribute types for 'other information'

This commit adds person attribute types for 'emergency contact' and 'matrimonial status' that are part of 'Other information' in the registration form.

tmvumbi2 committed:
COMRS-37 Internationalize labels for other information

This commit adds a translation for labels in 'Other information' in the registration form

lauravignoli committed:
Merge remote-tracking branch 'origin/dev' into COMRS-37

lauravignoli commented on configuration/concepts/registration_other_information.csv:
```diff
@@ -0,0 +1,22 @@
+Uuid,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Answers
```
@tmvumbi2 Is it necessary to add the uuids?

lauravignoli replied:
Oh I see that is useful to add them because some of those concepts are answers to other concepts.

lauravignoli committed:
COMRS-37 Update phoneNumber import file

tmvumbi2 committed:
COMRS-37 Fix translation of phone number

The result table (after searching for patient) uses personAttributeType.name as label (on the table's header). To have the header translated, personAttributeType.name needs to be the translation key.

lauravignoli committed:
COMRS-37 Update file name for local name person attr

Updated the name to follow the same name format that we used for other person attributes.

lauravignoli committed:
Merge remote-tracking branch 'origin/dev' into COMRS-37

# Conflicts:
#	configuration/personattributetypes/personAttributeTypes.csv
